### PR TITLE
[refactor] read team members from json file

### PR DESCRIPTION
**Pull request by vegetabill**
This is prep to add me to the team page. If it looks good we can do the same thing for the board members.

* [Before Screenshot](https://www.dropbox.com/s/j5wdbh1gwdayg0d/team-json-before.png?dl=0)
* [After Screenshot](https://www.dropbox.com/s/6en0jiujq3c4rhw/team-json-after.png?dl=0)

My only concern as a non-python dev is *will this JSON file be readable in a production env?*

### Small CSS Tweaks
  - use CSS for alternating left and right display
  - improve html semantics (avoid <br/> in heading)

### CODE REVIEW

File: main_site.py [hunk 1]
```diff
@@ -89,7 +90,9 @@ def render_team_page():
     """
     Renders the team page from jinja2 template
     """
-    return render_template("team.html")
+    with open('./team.json') as f:
```
Comment (vegetabill) [inline main_site.py]: Should we do this once at app boot? Not familiar with Flask lifecycle.

File: main_site.py [hunk 2]
```diff
@@ -5,6 +5,7 @@
 import os
 from dateutil.parser import parse
 import pendulum
+import json
```
Comment (gsong) [inline main_site.py]: Please use `isort` to sort imports.

File: main_site.py (hunk 1 above)
Comment (gsong) [reply main_site.py]: Since we're not concerned with performance, this is fine.

File: team.json [hunk 3]
```diff
@@ -0,0 +1,26 @@
+{
```
Comment (gsong) [inline team.json]: Can we please `prettier` this file?

File: team.json [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+        },
+        {
+            "name": "Tiffany Quach",
+            "link_url": "https://linkedin.com/in/tiffanyaquach/",
+            "photo_filename": "Tiffany-Quach-min.jpg",
+            "title": "Partnership Development Manager",
+            "blurb": "Tiffany has over 10 years of professional experience working in social impact. Prior to joining Techtonica, she managed a STEM and Environmental Education program for youth from communities that organizations have previously failed to engage. She is an experienced community builder and is impassioned about diversity, equity, and inclusion across all sectors. She is now excited to help create an inclusive tech ecosystem through partnership and relationship building."
+        }
+    ],
+    "board_members": []
```
Comment (gsong) [inline team.json]: Can we remove this until we actually need it?

File: templates/team.html [hunk 5]
```diff
@@ -7,57 +7,38 @@
...
-             alt="Judy Tuan"
-             class="resize-400px">
-      </div>
-      <div class="column centered">
-        <h2><a href="https://www.linkedin.com/in/judytuna/" target="_blank" rel="noopener noreferrer">Judy Tuan, Technical Apprenticeship Program Manager</a></h2>
-        <p class="justify">Judy Tuan became a software engineer through an apprenticeship at Blazing Cloud with the founder of RailsBridge. She has since worked as a software engineer and engineering manager at Indiegogo. During the last eight years, she has helped people join tech through her work with Learners Guild, RailsBridge, and Women Who Code. She currently volunteers with <a href="https://techworkerscoalition.org/" target="_blank" rel="noopener noreferrer">the Tech Workers Coalition</a>, which is a group building workers’ power.</p>
       </div>
-    </div> -->
-    </div>
+    {% endfor %}
```
Comment (gsong) [inline templates/team.html]: Remove trailing spaces.

File: templates/team.html [hunk 6]
```diff
@@ -7,57 +7,38 @@
...
+              alt="{{ employee['name'] }}"
+              class="resize-400px">
+        </div>
+        <div class="column centered">
+            <h2>
+              <a href="{{ employee['link_url'] }}" target="_blank" rel="noopener noreferrer">
+                {{ employee['name'] }}
+              </a>
+            </h2>
+            <h3>
```
Comment (gsong) [inline templates/team.html]: The title isn't a header, right?

File: templates/team.html [hunk 7]
```diff
@@ -7,57 +7,38 @@
...
+              <a href="{{ employee['link_url'] }}" target="_blank" rel="noopener noreferrer">
```
Comment (gsong) [inline templates/team.html]: Having two links to the same place next to each other is not good for accessibility.

File: templates/team.html [hunk 8]
```diff
@@ -7,57 +7,38 @@
...
+                {{ employee['title'] }}
+              </a>
+            </h3>
+            </a>
+            </h2>
```
Comment (gsong) [inline templates/team.html]: ???

**Comment by rogue0137** [comment]
**New thing**: I'm going to look through PRs every two weeks (ish). If there are questions, I'll try to address them/find the write person to address them. I'll also check out stuff locally to see if PRs are ready to merge. 
@vegetabill @gsong will leave this open for another 15 days; if not approved by then, I'll close the request.

**Comment by rogue0137** [comment]
@gsong noticing this is still not approved; should we close?

**Comment by gsong** [comment]
@rogue0137 Are there still plans to redo the website? If yes, I think you can safely close this. If not, it might be worthwhile cleaning it up a bit so updating the team page is less mistake-prone.

I wouldn't merge the code as-is.

**Comment by rogue0137** [comment]
@MichelleGlauser is the website being redone? also, what is entailed in a website "redo"?

**Comment by MichelleGlauser** [comment]
@rogue0137 There are no plans to re-do the website right now.

**Comment by rogue0137** [comment]
@MichelleGlauser sounds good!
@gsong I'll incorporate some of this into a PR I do later this week to update multiple bios

**Pull request closed without merging**